dipy/denoise/denspeed.pyx

```diff
@@ -69,17 +74,14 @@ def _nlmeans_3d(double [:, :, ::1] arr, double [:, :, ::1] mask,
         cnp.npy_intp i, j, k, I, J, K
         double [:, :, ::1] out = np.zeros_like(arr)
         double summ = 0
-        double sigm = 0
         cnp.npy_intp P = patch_radius
         cnp.npy_intp B = block_radius
 
-    sigm = sigma
```

Comment:
This was a single value before, now it's replaced by a whole array, hance the name change. See line 133 and 173 since this variable now pokes internally at the array instead of being "global".